js/src/bundle.js

```diff
@@ -258,19 +259,19 @@ var debug = true,
     // Move functions
 
     function movePlayer() {
-      if (rightKey) player.x += 5;
-      else if (leftKey) player.x -= 5;
-      if (upKey) player.y -= 5;
-      else if (downKey) player.y += 5;
-      if (player.x <= 0) player.x = 0;
-      if ((player.x + player.w) >= width) player.x = width - player.w;
-      if (player.y <= 0) player.y = 0;
-      if ((player.y + player.h) >= height) player.y = height - player.h;
+      if (rightKey) player_module.getPlayer().x += 5;
+      else if (leftKey) player_module.getPlayer().x -= 5;
+      if (upKey) player_module.getPlayer().y -= 5;
+      else if (downKey) player_module.getPlayer().y += 5;
+      if (player_module.getPlayer().x <= 0) player_module.getPlayer().x = 0;
+      if ((player_module.getPlayer().x + player_module.getPlayer().w) >= width) player_module.getPlayer().x = width - player_module.getPlayer().w;
+      if (player_module.getPlayer().y <= 0) player_module.getPlayer().y = 0;
+      if ((player_module.getPlayer().y + player_module.getPlayer().h) >= height) player_module.getPlayer().y = height - player_module.getPlayer().h;
     }
```

Comment:
If you're interested in making this a little more functional, you could call `movePlayer` straight from your keyboard event listener.  movePlayer would be called with the keyboard event, so now you don't have to use a variable in another scope to store the current key event:

```
function movePlayer(e) {
  if (e.keyCode === 38) ...
}
document.addEventListener('keydown', movePlayer)
```